src/main/java/io/divolte/server/filesinks/gcs/entities/GcsObjectResponse.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2017 GoDataDriven B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.divolte.server.filesinks.gcs.entities;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+
+public class GcsObjectResponse {
```

Comment:
Same as above. Here enforcing non-nullity is also not the most robust solution, as the upstream API could see changes that remove fields (they shouldn't allow that, but you never know). Many of these fields are not used, but just parsed for the purpose of better debug logging when enabled. It would be inconvenient to break on it.